plugins/gpgme-vala/src/gpgme_helper.vala

```diff
@@ -175,6 +175,9 @@ private static uint8[] get_uint8_from_data(Data data) {
 
 private static void initialize() {
     if (!initialized) {
+#if _WIN32
+        set_global_flag("w32-inst-dir", "C://Program Files (x86)//GnuPG//bin");
```

Comment:
A hard-coded path won't work if the installation differs from what you expect. If it isn't reasonably possible to automatically find the correct path on all systems, we'd need to allow users to enter their path manually.

Reply:
Uhm... what is the best way to handle this? Get the path from an environment variable, or add a textbox to the account config screen when OpenPGP is enabled?

Reply:
Do gpg4win installations set some environment variable? I've never used it, so I don't know what's the way to go. What do other apps do?

Reply:
Done